src/components/Container.js

```diff
@@ -172,6 +172,11 @@ export const Container = ({
 
   const handleScroll = useCallback(() => {
     if (!contentRef.current) return
+    const scrollableDistance =
+      contentRef.current.scrollHeight - contentRef.current.offsetHeight
+    if (scrollableDistance < 100) {
```

Comment:
nit: could extract into const

Reply:
good catch, it should come from the `scrollRange`